src/utils/i18n-utils.js

```diff
@@ -8,33 +8,44 @@
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-import appLocales from 'theme/lang/locales.json';
+import locales from 'theme/lang/locales.json';
 import { defaultLocale } from 'theme/lang/index.js';
 import { updateLangTag } from 'docc-render/utils/metadata';
 
-const localeCodes = new Set(appLocales.map(appLocale => appLocale.code));
+const slugsForLocale = locales.reduce((map, locale) => ({
+  ...map,
+  [locale.slug]: locale.code,
+}), {});
```

Comment:
I think it would be a little less confusing to name this `codeForLocale`? Especially since we are using it to get locale code from slug.

Reply:
I replaced `codeForLocale` with `codeForSlug`, you are right that it's easier to understand